server/chat-commands/room-settings.ts

```diff
@@ -730,6 +717,41 @@ export const commands: ChatCommands = {
 		`/subroomgroupchat [roomname] - Creates a subroom groupchat of the current room. Can only be used in a public room you have staff in.`,
 	],
 
+	async renamegroupchat(target, room, user) {
```

Comment:
Why is this a different command than `renameroom`, anyway?

Reply:
i figured it'd be clearer for some people.

Reply:
An alias would achieve that with less code duplication.

Reply:
Okay, then